editor/components/block-toolbar/style.scss

```diff
@@ -1,5 +1,5 @@
 .editor-block-toolbar {
-	display: inline-flex;
+	display: flex;
```

Comment:
I found that, as `inline-flex`, Firefox would add some height when rendered within the `display: table` wrapper. I see no reason why it needs to be `inline-flex`, as it's only ever rendered as the sole child in all of its usage.